extensions/ohif-i18n/src/locales/en/Buttons.json

```diff
@@ -0,0 +1,45 @@
+{
+  "Themes": "Themes",
+  "Previous": "$t(Common:Previous)",
+  "Next": "$t(Common:Next)",
+  "Play": "$t(Common:Play)",
+  "Stop": "$t(Common:Stop)",
+  "Layout": "$t(Common:Layout)",
+  "More": "$t(Common:More)",
+  "Crosshairs": "Crosshairs",
+  "Magnify": "Magnify",
+  "ROI Window": "ROI Window",
+  "Probe": "Probe",
+  "Ellipse": "Ellipse",
+  "Rectangle": "Rectangle",
+  "Invert": "Invert",
+  "Rotate Right": "Rotate Right",
+  "Flip H": "Flip H",
+  "Flip V": "Flip V",
+  "Clear": "Clear",
+  "Brush": "Brush",
+  "Coronal": "Coronal",
+  "Stack Scroll": "Stack Scroll",
+  "Measurements": "Measurements",
+  "Zoom": "Zoom",
+  "Levels": "Levels",
+  "Pan": "Pan",
+  "Length": "Length",
+  "Angle": "Angle",
+  "Bidirectional": "Bidirectional",
+  "Freehand": "Freehand",
+  "EllipticalRoi": "EllipticalRoi",
+  "CircleRoi": "CircleRoi",
+  "RectangleRoi": "RectangleRoi",
```

Comment:
Can you switch these to Ellipse, Circle, Rectangle (ROI is not necessary).

Reply:
Fixed